src/components/molecules/TimePickerInput/TimePickerInput.tsx

```diff
@@ -82,12 +85,22 @@ const TimeInput = forwardRef<HTMLInputElement, TimeInputProps>(
             placeholder: '0',
           })}
         />
-        <Clock
-          className={classNames(
-            classes.timeIcon,
-            disabled && classes.disabledIcon
-          )}
-        />
+        {icon ? (
```

Comment:
**[non-blocker]**

Could make this shorter + avoid inline conditional rendering by moving the conditional part to the "icon" prop passed here:

   <Icon
            icon={icon || Clock}
            className={classNames(
              classes.timeIcon,
              disabled && classes.disabledIcon
            )}
          />